rclcpp/include/rclcpp/generic_subscription.hpp

```diff
@@ -123,6 +123,32 @@ class GenericSubscription : public rclcpp::SubscriptionBase
   RCLCPP_PUBLIC
   void return_serialized_message(std::shared_ptr<rclcpp::SerializedMessage> & message) override;
 
+
+  // DYNAMIC TYPE ==================================================================================
+  // TODO(methylDragon): Reorder later
```

Comment:
What is this TODO about?

Reply:
I had meant to move the methods up next to their respective groups (e.g. create_dynamic_message with create_message/create_serialized_message), but I think separating out is nicer for now, so I'm going to remove the TODO